From 543a1ea665d2ceac955b8912dcda742f0f2ce280 Mon Sep 17 00:00:00 2001
From: Prarit Bhargava <prarit@redhat.com>
Date: Fri, 23 Jan 2026 19:04:17 -0500
Subject: [PATCH] common/hstore/scratch_test.go: Fix compile warnings

Fix these compile warnings

./scratch_test.go:73:46: (*testing.common).Errorf format %q has arg expected of wrong type []int
./scratch_test.go:83:46: (*testing.common).Errorf format %q has arg expected of wrong type []int

Signed-off-by: Prarit Bhargava <prarit@redhat.com>
---
 common/hstore/scratch_test.go | 4 ++--
 1 file changed, 2 insertions(+), 2 deletions(-)

diff --git a/common/hstore/scratch_test.go b/common/hstore/scratch_test.go
index 1f08d46a92ea..678893772efb 100644
--- a/common/hstore/scratch_test.go
+++ b/common/hstore/scratch_test.go
@@ -70,7 +70,7 @@ func TestScratchAddSlice(t *testing.T) {
 	expected := []int{1, 2, 3}
 
 	if !reflect.DeepEqual(expected, sl) {
-		t.Errorf("Slice difference, go %q expected %q", sl, expected)
+		t.Errorf("Slice difference, got %v expected %v", sl, expected)
 	}
 	_, err = scratch.Add("intSlice", []int{4, 5})
 
@@ -80,7 +80,7 @@ func TestScratchAddSlice(t *testing.T) {
 	expected = []int{1, 2, 3, 4, 5}
 
 	if !reflect.DeepEqual(expected, sl) {
-		t.Errorf("Slice difference, go %q expected %q", sl, expected)
+		t.Errorf("Slice difference, got %v expected %v", sl, expected)
 	}
 }
 
-- 
2.52.0

